api/messages.proto

```diff
@@ -34,10 +35,28 @@ message Ticket {
 
   // Properties contains custom info about the ticket.  Top level values can be
   // used in indexing and filtering to find tickets.
+  // TODO: Deprecate and remove this field.
   google.protobuf.Struct properties = 2;
 
   // Assignment associated with the Ticket.
   Assignment assignment = 3;
+
+  SearchFields search_fields = 4;
+
+  google.protobuf.Any mmf_input = 5;
```

Comment:
input / output suffixes are understood only in context of the original proposal discussion - i think their meaning is not obvious just as an Any field seen on the Ticket. 

If not Properties, can we simply call it data? Ticket-->data MatchProfiles-->data Assignment-->data ?

Lets discuss naming offline. The current input / output naming does not seem intuitive.

Reply:
Added a comment, willing to discuss further.

Reply:
Per discussion, renamed to extension.